config/logicsoft/ZDB5100.xml

```diff
@@ -0,0 +1,498 @@
+      <Help>        
+        Byte 1: Red brightness
+            0-255 Brightness level for the red colour in the 4 indicator groups (Default is 255).
+        Byte 2: Green brightness
+            0–255 Brightness level for the green colour in the 4 indicator groups (Default is 255).
+        Byte 3: Blue brightness
+            0-255 Brightness level for the blue colour in the 4 indicator groups (Default is 255).
+        Byte 4: Not used–must be set to 0.
+        Default: 4294967040
+			</Help>
+    </Value>
+    <Value genre="config" index="15" label="Associations groups, transmission when included secure" size="2" type="bitset" bitmask="8191" value="8191">
+      <Help>This parameter specifies if commands are transmitted as a secure message for each of the association groups.</Help>
+      <BitSet id="0">
+        <Label>Bit 0</Label>
+        <Help>All messages in all groups are sent as insecure.</Help>
+      </BitSet>
+      <BitSet id="1">
+        <Label>Bit 1</Label>
+        <Help>Messages in association group 2 are sent as secure.</Help>
+      </BitSet>
+      <BitSet id="2">
+        <Label>Bit 2</Label>
+        <Help>Messages in association group 3 are sent as secure.</Help>
+      </BitSet>
+      <BitSet id="4">
+        <Label>Bit 4</Label>
+        <Help>Messages in association group 4 are sent as secure.</Help>
+      </BitSet>
+      <BitSet id="8">
+        <Label>Bit 8</Label>
+        <Help>Messages in association group 5 are sent as secure.</Help>
+      </BitSet>
+      <BitSet id="16">
+        <Label>Bit 16</Label>
+        <Help>Messages in association group 6 are sent as secure.</Help>
+      </BitSet>
+      <BitSet id="32">
+        <Label>Bit 32</Label>
+        <Help>Messages in association group 7 are sent as secure.</Help>
+      </BitSet>
+      <BitSet id="64">
+        <Label>Bit 64</Label>
+        <Help>Messages in association group 8 are sent as secure.</Help>
+      </BitSet>
+      <BitSet id="128">
+        <Label>Bit 128</Label>
+        <Help>Messages in association group 9 are sent as secure.</Help>
+      </BitSet>
+      <BitSet id="256">
+        <Label>Bit 256</Label>
+        <Help>Messages in association group 10 are sent as secure.</Help>
+      </BitSet>
+      <BitSet id="512">
+        <Label>Bit 512</Label>
+        <Help>Messages in association group 11 are sent as secure.</Help>
+      </BitSet>
+      <BitSet id="1024">
+        <Label>Bit 1024</Label>
+        <Help>Messages in association group 12 are sent as secure.</Help>
+      </BitSet>
+      <BitSet id="2048">
+        <Label>Bit 2048</Label>
+        <Help>Messages in association group 13 are sent as secure.</Help>
+      </BitSet>
+      <BitSet id="4096">
+        <Label>Bit 4096</Label>
+        <Help>Messages in association group 14 are sent as secure.</Help>
+      </BitSet>
+      <BitSet id="8191">
```

Comment:
ehhh. These ID's are wrong. The ID = bit position, not the value. So it would be 1, 2, 3, 4, 5 etc etc. 

See aeotec/zw100.xml for a example.

Reply:
Thanks. This will be corrected in next commit.